README.md

```diff
@@ -1,9 +1,54 @@
-# scrappy
+# Scrappy the Slack Bot
 
 ![build](https://github.com/hackclub/scrappy/workflows/build/badge.svg)
 
-the slack bot that powers [scrapbook.hackclub.com](https://scrapbook.hackclub.com). to get started developing, ask @sampoder to be added to the `scrappy (dev)` app on Slack and for a `.env` file. then use `ngrok` to forward your local instance onto the world wide web.
+Scrappy is the Slack bot that powers [scrapbook.hackclub.com](https://scrapbook.hackclub.com). Scrappy automatically generates your Scrapbook and Scrapbook posts via Slack messages. For more information about how to sign up,
 
----
+[Click here to view the Scrapbook repository](https://github.com/hackclub/scrapbook), which hosts the Scrapbook web code.
 
-[learn more about how to get started](https://scrapbook.hackclub.com/about)
+## Commands
+Scrappy provides some helpful commands in Slack. These commands are also documented in our Slack if you send the message `/scrappy` in any channel.
+
+- `/scrappy-togglestreaks`: toggles your streak count on/off in your status
+- `/scrappy-togglestreaks all`: opts out of streaks completely
+- `/scrappy-open`: opens your scrapbook (or another user's if you specify a username)
+- `/scrappy-setcss`: adds a custom CSS file to your scrapbook profile. Check out this cool example!
+- `/scrappy-setdomain`: links a custom domain to your scrapbook profile, e.g. [https://zachlatta.com](https://zachlatta.com)
+- `/scrappy-setusername`: change your profile username
+- `/scrappy-setaudio`: links an audio file to your Scrapbook. [See an example here](https://scrapbook.hackclub.com/matthew)!
+- `/scrappy-setwebhook`: create a Scrappy Webhook we will make a blank fetch request to this URL every time you post
+- `/scrappy-webring`: adds or removes someone to your webring
+- *Remove* a post: delete the Slack message and Scrappy will automatically update for you
+- *Edit* a post: edit the Slack message and it will automatically update for you
```

Comment:
Might be worth adding that reacting with `:scrappy:` can add a post

Reply:
Such a good feature [94669b5](https://github.com/hackclub/scrappy/pull/317/commits/94669b5e46b6f191f45b4483aff07cbf3070b4f8)